deploy/charts/external-secrets/templates/rbac.yaml

```diff
@@ -82,6 +82,16 @@ rules:
     - "configmaps"
     verbs:
     - "create"
+{{- if .Values.leaderElect -}}
```

Comment:
Currently both `leaderelection` role and rolebinding are created whether leaderElect is enabled or disabled. If removing the rbac is desired this if should wrap both resources and not this individual permission.